build/common/scripts/setup-webhook.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+n=0
+try=10
+kubectl apply -f deploy/webhook.yaml
+
+while  [[ $? -ne 0 && $n -ne $try ]]
+do 
+    sleep 5
+    n=$((n+1))
+    echo "-----Webhook install failed, Please try again $n-----"
+    kubectl apply -f deploy/webhook.yaml
```

Comment:
Why do you need to continuously retry? What causes it to fail?

Reply:
Deploying webhook resources after cert-manager causes error. I guess cert-manager needs time to settle down. Instead sleep 10, multiple try is better way